core/include/moveit/task_constructor/stages/compute_ik.h

```diff
@@ -107,6 +107,10 @@ class ComputeIK : public WrapperBase {
 	void setMinSolutionDistance(double distance) {
 		setProperty("min_solution_distance", distance);
 	}
+
+	void setDefaultTimeout(double timeout) {
+		properties().property("timeout").setDefaultValue(timeout);
+	}
```

Comment:
I would propose to leave this function out of the interface. It's nothing the user should touch, they can always `setTimeout` if they want to.

Reply:
No problem. Already changed locally, need to cleanup my repo later...

Reply:
Done.